### PR TITLE
[PHP 8.1] Changelog entry for standard functions

### PR DESCRIPTION
**Pull request by saundefined**
- `current()`
- `end()`
- `key()`
- `next()`
- `prev()`
- `reset()`

Relates #1034

### CODE REVIEW

File: language-snippets.ent [hunk 1]
```diff
@@ -700,6 +700,16 @@ information. Use <link linkend="language.operators.comparison">the ===
 operator</link> for testing the return value of this
 function.</simpara></warning>'>
 
+<!-- Standard -->
+<!ENTITY standard.changelog.calling-on-objects '<row xmlns="http://docbook.org/ns/docbook">
+<entry>8.1.0</entry>
```
Comment (cmb69) [inline language-snippets.ent:705]:
```suggestion
 <entry>8.1.0</entry>
```

File: language-snippets.ent [hunk 2]
```diff
@@ -700,6 +700,16 @@ information. Use <link linkend="language.operators.comparison">the ===
...
+ <entry>
+  Calling on &object;s is deprecated.
```
Comment (cmb69) [inline language-snippets.ent:707]:
```suggestion
  Calling this function on &object;s is deprecated.
```

**Review by cmb69 — COMMENTED** [review]
Thank you! Besides the nits, this looks fine!

File: language-snippets.ent [hunk 3]
```diff
@@ -700,6 +700,16 @@ information. Use <link linkend="language.operators.comparison">the ===
 operator</link> for testing the return value of this
 function.</simpara></warning>'>
 
+<!-- Standard -->
+<!ENTITY standard.changelog.calling-on-objects '<row xmlns="http://docbook.org/ns/docbook">
+ <entry>8.1.0</entry>
+ <entry>
+  Calling this function on &object;s is deprecated.
+  Either use <function>get_mangled_object_vars</function> on the &object; first, or use <classname>ArrayIterator</classname>.
```
Comment (jrchamp) [inline language-snippets.ent:708]: This says to use ArrayIterator, but ArrayIterator doesn't work with these functions either. So then, what is this supposed to mean?

Comment (Girgias) [reply language-snippets.ent:708]: For the love of god, don't comment on a 2 year old PR, either open a new one or create an issue.

Nobody is going to act on this "review" comment.

**Pull request merged**